src/Elastic.Apm/Model/Span.cs

```diff
@@ -31,15 +29,16 @@ internal class Span : ISpan
 			IApmLogger logger
 		)
 		{
-			_start = DateTimeOffset.UtcNow;
+			Timestamp = TimeUtils.TimestampNow();
+			Id = RandomGenerator.GenerateRandomBytesAsString(new byte[8]);
+			_logger = logger?.Scoped($"{nameof(Span)}.{Id}");
```

Comment:
Is it a good idea to include the span id here? That's also logged as part of `Span.ToString()`, which I think we use everywhere:

```
[2019-07-03 02:33:43][Debug] - {Span.14aa6ce329bddc53} End() called on already ended Span{Id: 14aa6ce329bddc53, TransactionId: a857e56e2d6c1d78, ParentId: a857e56e2d6c1d78, TraceId: 8e063cef1f687d400c50db874609cdce, Name: PRAGMA foreign_keys=ON;, Type: db, IsSampled: True}. Start time: 2019-07-03 14:33:43.9529890 UTC (as timestamp: 1562164423952989), Duration: 3.0991ms
```
I think with this it'll be harder to filter only span related logs.

Reply:
The reason I thought it might useful to have Span ID in the category/logger name is this way we don't need to remember ti include `Span.ToString()` every time we print something from Span's context. 

> I think with this it'll be harder to filter only span related logs.

Maybe just a little bit harder
```
grep -e "{Span\.\o:h*}"
```
instead of
```
grep "{Span}"
```

You are - at the moment we don't print that much from Span's context and everything we print includes `Span.ToString()` - I just wanted to future proof it when for when add more logs to it.
If you think at the moment adding ID to category/logger does more harm than good I'll remove it.

Reply:
I just I was surprised by the logs, and it took me some seconds to realize that it's the span id - and realized this by seeing it again in the same lane later printed by `Span.ToString`.